journal.py~

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+from pyramid.config import Configurator
+from pyramid.view import view_config
+from waitress import serve
+
+
+@view_config(route_name='home', renderer='string')
+def home(request):
+    return "Hello World"
+
+
+def main():
+    """Create a configured wsgi app"""
+    settings = {}
+    debug = os.environ.get('DEBUG', True)
+    settings['reload_all'] = debug
+    settings['debug_all'] = debug
+    # configuration setup
+    config = Configurator(
+        settings=settings
+    )
+    config.add_route('home', '/')
+    config.scan()
+    app = config.make_wsgi_app()
+    return app
+
+
+if __name__ == '__main__':
+    app = main()
+    port = os.environ.get('PORT', 5000)
+    serve(app, host='0.0.0.0', port=port)
+
```

Comment:
Again, this entire file is something that shouldn't be here.  See above about ignoring `*~`